addons/mass_mailing/__init__.py

```diff
@@ -5,3 +5,20 @@
 from . import models
 from . import report
 from . import wizard
+
+from odoo import api, SUPERUSER_ID
+
+
+def uninstall_hook(cr, registry):
+    # restore action to send simple mails in mass mode
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env.ref('mail.action_partner_mass_mail').write({
+        'res_model': 'mail.compose.message',
```

Comment:
I woudl also force the name update to 'Send email', in case some people renamed it differently